sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -57,9 +57,16 @@ class Mean(object):
 
   class Globally(ptransform.PTransform):
     """combiners.Mean.Globally computes the arithmetic mean of the elements."""
+    def __init__(self, as_view=False, has_defaults=True):
```

Comment:
This file is unrelated to the examples and has API changes. What is the reason for these changes? Please separate them to another PR.